qiita_db/support_files/patches/26.sql

```diff
@@ -0,0 +1,52 @@
+-- June 11,, 2015
+-- Adds portal type to analyses
+
+-- Remove existing portals and replace with more relevant ones
+UPDATE qiita.study SET portal_type_id = 1 WHERE portal_type_id = 3;
```

Comment:
This is actually incorrect. If a study previously belonged to the `QIIME_EMP` portal, means that now it belongs to the EMP portal (because all of them belong to the QIITA portal by default):
```sql
UPDATE qiita.study SET portal_type_id = 2 WHERE portal_type_id = 3;
```

Also, the comment is misleading, as this line is not replacing the portal with relevant ones.